www/themes/upd/templates/node/node--landing-page.html.twig

```diff
@@ -13,7 +13,12 @@
       </div>
     </div>
     <div class="wrapper wrapper--gutter">
+{% if content.field_standfirst|render|trim %}
+      <h1 class="hero__title with-sub-title">{{ label }}</h1>
+      <div class="hero__sub-title">{{ content.field_standfirst }}</div>
+    {% else %}
       <h1 class="hero__title">{{ label }}</h1>
+    {% endif %}
```

Comment:
A very minor niggle, could you fix the `{% endif %}` indent so it lines up with `{% if content.field_standfirst|render|trim %}`?

Just makes it easier to visually parse 😃